File: SoloLearn Codes/Neverland/Neverland.cs
using System;


namespace Sololearn
{
    class Program
    {
        static void Main(string[] args)
        {
            int age = Convert.ToInt32(Console.ReadLine());
            int year = Convert.ToInt32(Console.ReadLine());
            Console.WriteLine("My twin is " + (age + year) + " years old and they are " + year + " years older than me");
        }
    }
}

File: SoloLearn Codes/Neverland/Neverland.java
import java.util.Scanner;

public class Neverland
{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int age = sc.nextInt();
        int year = sc.nextInt();
        System.out.println("My twin is " + (age + year) + " years old and they are " + year + " years older than me");
	}
}